### DISCUSSION TITLE
Uppercase and lowercase values creates 2 different tags

### DISCUSSION THREAD

**Discussion by b4oshany** [spatie/laravel-tags#446] [General]
Please see the screenshot with the test I wrote

<img width="548" alt="image" src="https://user-images.githubusercontent.com/1830494/217201133-59f9dc0e-24f1-498f-98c7-68d141241298.png">

Yes, I'm aware that I can simple force everything to be stored in lowercase, but I thought you should be aware of this issue.

**Comment by b4oshany** [comment]
Here's a screenshot fo the db table:
![image](https://user-images.githubusercontent.com/1830494/217203826-1d7fda65-7e91-4f5b-9fe5-11ff930261b3.png)

Notice that the slug is the same for the tag "first user", yet it yields additional records in the database. I checked the code and noticed that the `HasTag::attachTag` method eventually calls the `Tag::findFromString` static function. The logic seems fine, but it's giving the incorrect result.

**Comment by Rattone** [comment]
Same problem here, also identical slugs are created, which are supposed to be unique
